cylc/flow/etc/tutorial/runtime-tutorial/flow.cylc

```diff
@@ -30,5 +30,9 @@
         """
 
 [runtime]
+    [[consolidate_observations]]
+    [[forecast]]
+    [[get_rainfall]]
+    [[post_process_exeter]]
```

Comment:
Adding these in, I still get an error validating the workflow in [step 2](https://cylc.github.io/cylc-doc/stable/html/tutorial/runtime/runtime-configuration.html#admonition-2).

```
WorkflowConfigError: implicit tasks detected (no entry under [runtime]):
    * get_observations_aldergrove
    * get_observations_shetland
    * get_observations_camborne
    * get_observations_heathrow
To allow implicit tasks, use 'flow.cylc[scheduler]allow implicit tasks'
```